recipes/transdecoder/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: transdecoder
+  version: "2.1.0"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: transdecoder-2.1.0.tar.gz
+  url: https://github.com/TransDecoder/TransDecoder/archive/v2.1.0.tar.gz
+  md5: f86c688e13ced3cc51fed3d508a2ea05
+
+requirements:
+  build:
+    - gcc
```

Comment:
@johanneskoester There is no `gcc` version that satisfies in `>= 4.9.1` in the current channels, even if there is `libgcc 5.2.0`. This seems to work fine as as now.